hypertrace-gradle-docker-plugin/build.gradle.kts

```diff
@@ -4,8 +4,8 @@ plugins {
 }
 
 java {
-  targetCompatibility = JavaVersion.VERSION_1_8
-  sourceCompatibility = JavaVersion.VERSION_1_8
+  targetCompatibility = JavaVersion.VERSION_11
```

Comment:
Not strictly related but this plugin has some java 11 references in it so although generating jre8 classes it would have failed at runtime. We don't need the 8 support so fixed the compatibility rather than the code